### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by dmontoya1**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by dmontoya-servi — COMMENTED** [review]
All Ok, can merge

**Review by dmontoya1 — COMMENTED** [review]
All okay, can you nmergeplease

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (dmontoya1) [inline index.html:50]:
```suggestion
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment (dmontoya1) [inline index.html]:
```suggestion
<h2 hidden>Game over, refresh to play againnn 🧑‍💻 🤖!</h2>
```

**Pull request merged**